docs/src/examples/spindle.adoc

```diff
@@ -129,7 +128,7 @@ the man pages, just say 'man limit2' or 'man near' in a terminal.
 
 [source,{hal}]
 ----
-# load real time a limit2 and a near with names so it is easier to follow
+# instantiate the real time modules limit2 and near with names so it is easier to follow their connections
```

Comment:
Noy sure if 'instantiate' is the right word here.

Reply:
I liked it. It is used http://linuxcnc.org/docs/stable/html/gui/qtvcp_libraries.html in the documentation already. Same context? I mean, you want to give that hal component a name to refer to it as an individual and trace its connections, "instantiate" is a better fit than "load".

Reply:
But the corresponding command is named `loadrt` ;-)
And the context in [qtvcp_libraries.html](http://linuxcnc.org/docs/stable/html/gui/qtvcp_libraries.html) is python code, not HAL commands.
Sorry for bothering you with my comments as the first action when I'm back home :smile:

Reply:
The "load" everyone can read in the line below. I still think to make a proper comment. And I still like it.

Reply:
We usually use 'load' for HAL components.
Instantiate is commonly used in python.

A typical machinist type user would probably be confused by instantiate.